src/remotestorage.js

```diff
@@ -703,16 +703,18 @@ RemoteStorage.prototype = {
    * this is mostly useful for letting users sync manually, when pressing a
    * sync button for example. This might feel safer to them sometimes, esp.
    * when shifting between offline and online a lot.
+   *
+   * @returns {Promise} A Promise which resolves when the sync has finished
    */
   startSync: function () {
     if (!config.cache) { return; }
     this.sync.stopped = false;
     this.syncStopped = false;
-    this.sync.sync();
+    return this.sync.sync();
   },
 
   /**
-   * TODO: document
+   * Stop the synchronization interval.
```

Comment:
One can't "stop" an interval really. Is this referring to stopping the periodic sync altogether?

Reply:
Yes.